packages/core/src/configuration.ts

```diff
@@ -108,15 +113,35 @@ export const barChartTooltip: BarTooltipOptions = Tools.merge(
 const axes: AxesOptions = {
 	top: {
 		includeZero: true,
+		truncation: {
```

Comment:
maybe we can store this in a variable? `standardTruncationOptions`?

Reply:
Resolved. Thanks.